polynote-kernel/src/main/scala/polynote/kernel/lang/python/PythonInterpreter.scala

```diff
@@ -310,7 +298,7 @@ class PythonInterpreter(val symbolTable: RuntimeSymbolTable) extends LanguageInt
       }
       jep.set("__polynote_code__", code)
       jep.eval("import jedi")
-      jep.eval(s"__polynote_cmp__ = jedi.Interpreter(__polynote_code__, [globals(), locals()], line=$line, column=$col).completions()")
+      jep.eval(s"__polynote_cmp__ = jedi.Interpreter(__polynote_code__, [__polynote_globals__, {}], line=$line, column=$col).completions()")
```

Comment:
I guess `locals()` was always blank anyways?

Reply:
Yes, we always use a blank locals dict. When running code, we pass in a new blank locals dict as a variable so we can get out the newly defined values from it, but here that's unnecessary since nothing will run.